client/my-sites/plans/utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+
+import { abtest } from 'lib/abtest';
+import { getSavedVariations } from 'lib/abtest';
+
+function isInFreeTrialTest() {
+	if ( getSavedVariations().freeTrials_20160107 && abtest( 'freeTrials' ) === 'offered' ) {
```

Comment:
I see... because we want the test to run only for users who just went through the signup?
Could we just test for `getABTestVariation( 'freeTrials' )`? It looks cleaner to me.